{kokoro} Fix job that checks for source files not covered by a BUILD target

romoore opened the pull request:
The job was broken due to a few things:
* It did not update bazel on the kokoro runner after install.
* It used the wrong bazel query to determine if a file was included in a BUILD package.
* It was not posting comments to the PR, forcing authors to hunt through the logs (which not all authors can access).

Commit eaa9d12554a2d22275124bd7e34c48fab148b0b4 (in this PR's history) originally included several files to test the behavior of the job. Since I cannot turn the job on for the `develop` branch until these changes are made (because it will fail with spurious errors), the testing took place in this branch independently.

Commit:
Script for uncovered files

Commit:
Testing uncovered job

Commit:
[scripts] Using git query to find affected files

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Adding new file that I forgot to put in bazel

Commit:
Moving into git repository first

Commit:
Versioning, pushd

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
little cleanup

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Few escapes

Commit:
Debugging messages

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Debugging more

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
More debug statements

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Debug, please

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
too many quotes

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
debug branches

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Reference origin instead of local branch

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Got a sha, removing echoes

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Removing more echoes

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Fixing array expansion

Commit:
2 more files

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Merge branch 'develop' into bazel-uncovered

Commit:
Upgrade bazel to 0.20.0 before running queries.

Commit:
Merge branch 'develop' into test-uncovered

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Adding new file with '+' in name to test latest bazel

Commit:
Integrating github comment posting

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Echoing GitHub message in logs when running on kokoro

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Adding debug list of changed files

Commit:
Debugging comments script

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
More debugging of github comments

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Installing bazel earlier in the script

Commit:
* Only upgrade bazel if files change

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
* Update github-comments repo
* Debugging swift version

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Refactoring select_xcode to a library script

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Only execute `select_xcode` if XCODE_VERSION is defined

Commit:
Fixing select_xcode invocation

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Bringing in `version_as_number()`

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Debugging bazel invocation

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Enter bazel workspace before running `query`

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Removing raw bazel invocation, since it makes scripts fail

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

Commit:
Turn off exit-on-error

Commit:
Adding file covered by BUILD targets

material-automation commented:
The following files are not in any BUILD targets:

```
components/AppBar/src/MaterialAppBar.h
components/AppBar/src/Tester/MaterialAppBar+Tester.h
file.swift
foo.m
```


<!-- identifier: bazel-rule-coverage -->

Commit:
Fixing bazel query.

Previous query incorrectly identified some files as not covered by a `rule`.
Instead, we just want to know the package (if any) a file belongs to.

Commit:
Merge branch 'bazel-uncovered' into test-uncovered

material-automation commented:
The following files are not in any BUILD targets:

```
components/AppBar/src/Tester/MaterialAppBar+Tester.h
file.swift
foo.m
```


<!-- identifier: bazel-rule-coverage -->

Commit:
Reverting test-related changes

material-automation commented:
bazel detected changes to the following targets:

```
//components/AppBar:unit_tests
//components/AppBar:unit_tests_IPAD_PRO_12_9_IN_9_3
//components/AppBar:unit_tests_IPAD_PRO_12_9_IN_9_3_test_bundle
//components/AppBar:unit_tests_IPHONE_5_IN_8_1
//components/AppBar:unit_tests_IPHONE_5_IN_8_1_test_bundle
//components/AppBar:unit_tests_IPHONE_7_PLUS_IN_10_3
//components/AppBar:unit_tests_IPHONE_7_PLUS_IN_10_3_test_bundle
//components/AppBar:unit_tests_IPHONE_X_IN_11_0
//components/AppBar:unit_tests_IPHONE_X_IN_11_0_test_bundle
```

<!-- identifier: affected-targets -->

Commit:
Removing space
